### PR TITLE
docs: Add info to FAQ about SSH keys and node access

### PR DESCRIPTION
**Pull request by chrisnegus**
Fixes: Issue [#2432](https://github.com/aws/karpenter/issues/2432), [#1807](https://github.com/aws/karpenter/issues/1807), and [#2269](https://github.com/aws/karpenter/issues/2269).

**Description**
There have been requests to be able to add SSH keys to instances used for Karpenter nodes, via provisioners or similar means. Because the Karpenter project prefers not to add SSH keys via provisioners, this PR recommends alternate ways to access Karpenter nodes, but also describes how to use userData in custom launch templates to add SSH keys when those alternate ways are not suitable.

**Does this change impact docs?**
- [X] Yes, PR includes docs updates
- [ ] Yes, issue opened: # 
- [ ] No

### CODE REVIEW

**Review by tzneal — APPROVED** [review]
lgtm

File: website/content/en/preview/faq.md [hunk 1]
```diff
@@ -84,6 +84,33 @@ If this were to occur, a node could remain non-empty and have its lifetime exten
...
+MIME-Version: 1.0
+Content-Type: multipart/mixed; boundary="BOUNDARY"
+--BOUNDARY
+Content-Type: text/x-shellscript; charset="us-ascii"
+
+#!/bin/bash
+mkdir -p ~ec2-user/.ssh/
+touch ~ec2-user/.ssh/authorized_keys
+cat >> ~ec2-user/.ssh/authorized_keys <<EOF
+{{ insertFile "../my-authorized_keys" | indent 4  }}
```
Comment (jonathan-innis) [inline website/content/en/preview/faq.md:107]: Is this a secure way to pass in a private SSH key? I believe User Data is unencrypted from the EC2 service-side so not sure if this is the recommended way to inject as private key into an instance.

**Review by jonathan-innis — CHANGES_REQUESTED** [review]

File: website/content/en/preview/faq.md (hunk 1 above)
Comment (tzneal) [reply website/content/en/preview/faq.md:107]: The public key goes in the authorized_keys files, while the private key remains on the client you are SSH'ing from.

Comment (jonathan-innis) [reply website/content/en/preview/faq.md:107]: 🤦‍♂️

**Review by jonathan-innis — APPROVED** [review]
🚀

File: website/content/en/preview/faq.md (hunk 1 above)
Comment (chrisnegus) [reply website/content/en/preview/faq.md:107]: Looks like this already merged, but I'm happy to change it if you can suggest another way.

**Pull request merged**